pretty_gpx/common/data/overpass_processing.py

```diff
@@ -68,7 +67,8 @@ def simplify_ways(coordinates: list[ListLonLat],
     """Simplify a list of ways using Douglas-Peucker algorithm from shapely."""
     tolerance = np.rad2deg(tolerance_m/EARTH_RADIUS_M)
     total_hausdorff_distance = 0
-    coordinates = merge_ways(coordinates, eps=tolerance, verbose=False)
+    logger.debug("Merge ways")
+    coordinates = merge_ways(coordinates, eps=tolerance, verbose=DEBUG_DISTANCE)
```

Comment:
@JulesL2 
the tolereance should vary with the papersize , no ?
the same street polyline should be drawn with less accuracy when used on a large marathon map compared to a small 5k run around this area

Reply:
We can indeed separate the merging tolerance which should stay between 1 and 5m for all paper sizes and the tolerance for the simplify function that will depend on the paper size.

Reply:
Moreover as we are building a svg file, it is a nonsense to remove details that are visible with a reasonnable zoom, there is pdf export if we want to remove even small details. 
As 5m is around the width of roads when they are drawn, we could do 10m but 50m or 100m is way too large, we need to remove those details instead of trying to simplify them